diff -dur -x '*~' Zope-3.4.0.orig/Dependencies/zope.app.server-Zope-3.4.0/zope.app.server/mkzopeinstance.py Zope-3.4.0/Dependencies/zope.app.server-Zope-3.4.0/zope.app.server/mkzopeinstance.py
--- Zope-3.4.0.orig/Dependencies/zope.app.server-Zope-3.4.0/zope.app.server/mkzopeinstance.py	2005-08-02 20:01:20.000000000 +0200
+++ Zope-3.4.0/Dependencies/zope.app.server-Zope-3.4.0/zope.app.server/mkzopeinstance.py	2005-08-30 12:14:12.000000000 +0200
@@ -333,7 +333,7 @@
         options.add_package_includes = from_checkout
         basedir = os.path.dirname(path)
         # no assurance that this exists!
-        options.skeleton = os.path.join(basedir, "zopeskel")
+        options.skeleton = "/etc/zope3/zopeskel"
     else:
         options.add_package_includes = False
     options.program = prog
